src/components/ReportActionItem/TaskPreview.js

```diff
@@ -88,9 +88,9 @@ function TaskPreview(props) {
                         disabled={ReportUtils.isCanceledTaskReport(props.taskReport)}
                         onPress={() => {
                             if (isTaskCompleted) {
-                                Task.reopenTask(props.taskReportID, taskTitle);
+                                Task.reopenTask(props.taskReport, taskTitle);
```

Comment:
Please double check that we are replacing this in all occurrences of `Task.reopenTask` and `Task.completeTask`. It seems we are missing some cases e.g. `TaskHeaderActionButton` and `HeaderView`

Reply:
done

Reply:
@s-alves10 Please re-check this

Reply:
@s77rt 
I'm sorry. Now fixed